### ISSUE TITLE
[Feat]: Make a release of WebDriverAgent with new functionality that was implemented for AccessibilityAudit and others

### ISSUE THREAD

**Issue by IlyaUA** [appium/appium#19445]
Release WebDriverAgent to unlock the ability to add documentation with the new accessibility audit method output for appium-xcui-driver
**Issue**
[[Feat]: WebDriverAgent, enhancing the /performAccessibilityAudit method to include detailed XCUIElement information](https://github.com/appium/appium/issues/19354)
**Implementation**
https://github.com/appium/WebDriverAgent/pull/808

**Comment by mykola-mokhnach** [comment]
there is no need to create a separate issue for this. Releases of most Appium repositories are being done automatically based on conventional commits. For example, your change to WDA has been already published and the latest version of it is 5.15.1 as of now (this could be verified in the [commits list](https://github.com/appium/WebDriverAgent/commits/master) )

Dependant usually takes care about picking up updated dependencies, but it is sometimes slow as many other packages are being updated. You may speed up this process by simply bumping the WDA version in the package.json of appium-xcuitest-driver and updating package.lock+the documentation on the added features. Check, for example, https://github.com/appium/appium-xcuitest-driver/pull/2171

**Comment by IlyaUA** [comment]
@mykola-mokhnach I tried and I get an error
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for appium-webdriveragent@^5.15.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

**Comment by mykola-mokhnach** [comment]
yes, It is seems like the release script does not do well. I've published the project manually to unblock you for now
